compiler/src/dmd/expression.d

```diff
@@ -3576,6 +3577,8 @@ extern (C++) final class NewExp : Expression
     bool onstack;               // allocate on stack
     bool thrownew;              // this NewExp is the expression of a ThrowStatement
 
+    Expression lowering;        // lowered druntime hook: `_d_newclass`
```

Comment:
Is this necessary? If yes keep it but keep an eye on adding unnecessary fields as dmd's classes are absolutely enormous